### PR TITLE
Bug fixes: trial by fire

### LINKED ISSUE #901: Aliens can pilot exosuits
As it says, Aliens can pilot exosuits while not being able to pick up and fire a gun, probably other things too, mayhaps make it so that the same flag that stops creatures from firing guns stops them from piloting suits as well.

### LINKED ISSUE #1215: The AI can unload trays by alt-clicking

### LINKED ISSUE #1597: Fletchette rifle doesn't actually work with the wield-rifle verb
While having the rifle makes the Wield rifle verb appear in your Object tab, trying to use that verb doesn't actually work, you have to click the top left icon to hold the Fletchette gun in both hands.

### LINKED ISSUE #1854: Players that are admin winded aren't notified they've been admin winded
I got (accidentally) winded and I didn't get any kind of notification for it. All I saw was I suddenly passed out asleep with no message indicating why.

### LINKED ISSUE #3216: Staff of Change is Broken.
The staff of change seems to be broken. All it does it make you various humans, a slime, or a robot. You can become a Diona but you're still just a human with diona abilities avalible.

### LINKED ISSUE #3567: You can dual weild a fire axe if it's impailed in your body.
also brings back the good ol' mining bug where you could throw offhands.

### LINKED ISSUE #3681: Wielding Melee Items on Back
Most probably related to #3567 

Having a twohanded melee item like a fire axe or baseball bat on your back still allows you to use the wield two-handed weapon verb. You can use the verb again and it will go back to as it was before without any issues, but if you attempt to wield it while it's on your back, and THEN equip it in an active hand, it gets slightly funky where it still counts as a onehanded weapon and your other hand still has the offhand. Attempting to use the wield weapon verb again does not do anything, and the only way to get rid of the offhand is to switch to its active hand that is holding the offhand and drop it.

### LINKED ISSUE #3913: Smelting glass shard in hand doesn't give glass
As name states, smelting glass shard while holing it in your hand will not give you glass, the shard just goes away.

### LINKED ISSUE #4469: Nymphs still play Follow the Leader while dead.
Watched a diona atmos tech get mauled by forty spiders in the bar's backroom while the last surviving nymph escaped and reformed. When the door opened the nymph corpses glided out of the door and started following the reformed diona until it absorbed them.

### LINKED ISSUE #4697: You can open/close laptops from any range.
Alt+clicking a laptop from **ANY** range will open/close it.

### PR DESCRIPTION
**Pull request by Alberyk**
-fixes #4697
-fixes #3913
-fixes #3216
-fixes #1854
-fixes #901
-fixes #1597
-fixes #4469
-fixes #1215
-fixes #3681
-fixes #3567

### CODE REVIEW

File: code/modules/mob/living/carbon/alien/diona/life.dm [hunk 1]
```diff
@@ -40,7 +40,8 @@
 /mob/living/carbon/alien/diona/think()
 	..()
 	if (!gestalt)
-		if(master_nymph && !client && master_nymph != src)
-			walk_to(src,master_nymph,1,movement_delay())
-		else
-			walk_to(src,0)
+		if(!stat == DEAD)
```
Comment (skull132) [inline code/modules/mob/living/carbon/alien/diona/life.dm]: erm. Shouldn't this be `stat != DEAD`?

File: code/modules/modular_computers/computers/subtypes/dev_laptop.dm [hunk 2]
```diff
@@ -16,6 +16,7 @@
 	var/icon_state_closed = "laptop-closed"
 
 /obj/item/modular_computer/laptop/AltClick()
+	if(!usr || usr.stat || usr.lying || usr.restrained() || !Adjacent(usr))	return
```
Comment (skull132) [inline code/modules/modular_computers/computers/subtypes/dev_laptop.dm]: `if (!use_check(usr, FALSE)) return`

File: code/game/objects/items/weapons/trays.dm [hunk 3]
```diff
@@ -214,9 +214,9 @@
 	return 1
 
 /obj/item/weapon/tray/AltClick(var/mob/user)
+	if (!use_check(user, FALSE)) return
```
Comment (skull132) [inline code/game/objects/items/weapons/trays.dm]: shit I fucked up. This should be `if (!use_check(user, show_messages = FALSE)) return`

File: code/modules/mob/living/carbon/alien/diona/life.dm [hunk 4]
```diff
@@ -40,7 +40,8 @@
 /mob/living/carbon/alien/diona/think()
 	..()
 	if (!gestalt)
-		if(master_nymph && !client && master_nymph != src)
-			walk_to(src,master_nymph,1,movement_delay())
-		else
-			walk_to(src,0)
+		if(!stat != DEAD)
```
Comment (skull132) [inline code/modules/mob/living/carbon/alien/diona/life.dm]: Remove the leading ! here. It should just be `if (stat != DEAD)`

File: code/game/mecha/mecha.dm [hunk 5]
```diff
@@ -1184,6 +1184,8 @@
 		if(C.handcuffed)
 			usr << "<span class='danger'>Kinda hard to climb in while handcuffed don't you think?</span>"
```
Comment (Arrow768) [inline code/game/mecha/mecha.dm]: Can you fix that message aswell ?
"while in handcuffs" or "while handcuffed"

Comment (Arrow768) [reply code/game/mecha/mecha.dm]: Disregard.

**Pull request merged**